content/html/concepts/quotations/quotations.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Quotations'
+Description: 'In HTML, a quotation is a portion of text that is attributed to someone else. '
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Quotations'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In HTML, a quotation is a portion of text that is attributed to someone else. HTML provides several ways to mark up quotations on a web page, using specific [tags](https://www.codecademy.com/resources/docs/html/tags) for short and long quotations.
+
+## Short Quotations
+
+Short quotations are used for brief, inline quotations that are a few words or less. To create a short quotation, you can use the [`<q>` tag](https://www.codecademy.com/resources/docs/html/elements/q). Here's an example of how to use the `<q>` tag:
+
+```pseudo
+<p>According to Albert Einstein, <q>Imagination is more important than knowledge.</q></p>
+```
+
+In this example, the text "Imagination is more important than knowledge" is enclosed in the `<q>` tag to indicate that it's a quotation. The quotation marks around "Imagination is more important than knowledge" are automatically added by the browser.
+
+## Long Quotations
+
+Long quotations are used for quotations that are longer than a few words and typically require their own paragraph. To create a long quotation, you can use the `[<blockquote>](https://www.codecademy.com/resources/docs/html/elements/blockquote)` tag. Here's an example of how to use the `<blockquote>` tag:
+
+```pseudo
+<blockquote>
+<p>"The best way to predict your future is to create it."</p>
```

Comment:
I think it would be preferable to show something that's a bit longer. This quote is almost the same length as the previous example